docs/decisions/0016_automatic_expiration.rst

```diff
@@ -0,0 +1,55 @@
+0016 Automatic Cancellation (Expiration)
+****************************************
+
+Status
+======
+Accepted - January 2024
+
+Context
+=======
+
+How expiration currently works
+------------------------------
+There's currently a management command (run via a cron) named ``automatically_exire_assignments.py``
+that toggles the state of ``LearnerContentAssignment`` records ("Assignments")
+to ``CANCELLED`` under any of the following conditions:
+
+1. The current state of the assignment is ``ALLOCATED``.
+2. The current date is more than 90 days after the most recent notification action - this action
+   time is used as a proxy for understanding *when* the assignment *was last allocated*. This helps
+   deal with the edge case where an assignment is allocated, cancelled, and re-allocated later. Note
+   that a reminder action on an assignment record *does not* reset this specific expiry time.
+3. The current date is greater than the inferred enrollment deadline for the assigned course.
+4. The current date is greater than the expiration date of the subsidy associated with the Assignment's
+   access policy record.
+
+Decision
+========
+The above management command will now remove Personally-Identifiable Information ("PII") from assignments
+that are automatically moved from ``ALLOCATED`` to ``CANCELLED`` under condition (1) above - that is, only
+for such assignments whose last notification time was more than 90 days ago.
+This PII includes the learner email address and their ``lms_user_id``.
```

Comment:
"lms_user_id is not PII" was my operating assumption for the last 5 years

Reply:
Oh hmm, maybe we got it wrong in license-manager?  That existing prior art is the only reason I decided to clear it here.